### PR TITLE
Merging collapsible if statements increases the code's readability

### PR DESCRIPTION
**Pull request by AymanDF**
This pull request is focused on resolving occurrences of Sonar rule squid:S1066 - “Collapsible "if" statements should be merged”. You can find more information about the issue here: 
https://dev.eclipse.org/sonar/rules/show/squid:S1066
Please let me know if you have any questions.
Ayman Abdelghany.

### CODE REVIEW

**Comment by metc** [comment]
Hi @AymanDF 
Could you please rebase with the `1.7.10-MNA` branch ?
I fixed our CI system today. We would like to do some checks before merging. Thanks !

File: src/main/java/mods/eln/gui/GuiTextFieldEln.java [hunk 1]
```diff
@@ -83,12 +83,10 @@ public boolean getEnabled() {
...
     public boolean textboxKeyTyped(char par1, int par2) {
-        if (getEnabled() && this.isFocused()) {
-        	if(par1 == '\r') {
-        		setFocused(false);
-        		return true;
-        	}
-        }
+        if (getEnabled() && this.isFocused() && par1 == '\r') {
+			return true;
+			setFocused(false);
```
Comment (metc) [inline src/main/java/mods/eln/gui/GuiTextFieldEln.java:87]: Should be inverted

Comment (AymanDF) [inline src/main/java/mods/eln/gui/GuiTextFieldEln.java:87]: Please accept my apologies for this accidental mistake. 
It has been fixed.

**Comment by metc** [comment]
Sounds good, @Electrical-Age/devs please review

**Pull request merged**